### PR TITLE
Don't show full block info in info message.

### LINKED ISSUE #2946: JSON logs of block information is not pretty printed
## Abstract

When running Raiden in the Devcon 4 Workshop https://github.com/raiden-network/workshop-devcon4/#running-raiden, when the node finishes launching it prints JSON block information inline as shown the in the screenshot below.
screen shot 2018-11-01 at 13 39 16

It would be much more developer-friendly (easier to read the information) if the JSON data was pretty printed

### PR DESCRIPTION
**Pull request by palango**
Fixes #2946 

I think the whole block info doesn't help with anything.

We might also want to lower this to debug level.

### CODE REVIEW

**Comment by ulope** [comment]
Note: This was changed from pretty much what you propose to include the full block only about a month ago in c3cd780ea525daaa1771ff7d850b9926dd1783e9

**Comment by palango** [comment]
@hackaugusto Do we require  the full block info? From my view we could always get this from the block hash.

**Comment by hackaugusto** [comment]
@palango This was introduced to debug reorg/gasprice problems, but introducing that much information on an info log was probably a mistake. We should be fine with the data we have in the Block state change.

File: raiden/tasks.py [hunk 1]
```diff
@@ -145,19 +145,21 @@ def loop_until_stop(self):
...
+                    block_hash=latest_block_hash,
                 )
 
                 if latest_block_number > last_block_number + 1:
                     missed_blocks = latest_block_number - last_block_number - 1
                     log.info(
                         'Missed block(s)',
                         missed_blocks=missed_blocks,
-                        latest_block=latest_block,
+                        latest_block=latest_block_number,
```
Comment (LefterisJP) [inline raiden/tasks.py]: nitpick: /s/latest_block/latest_block_number

**Comment by palango** [comment]
> We should be fine with the data we have in the Block state change.

But then lets remove the whole info block (as the user can't do anything anyways) and add the number of missed blocks in the debug call if it is bigger than 1.

**Pull request merged**